apps/auth/views.py

```diff
@@ -0,0 +1,41 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth import logout as log_out
+from django.conf import settings
+from django.http import HttpResponseRedirect
+from urllib.parse import urlencode
+
+import json
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "import json" should be placed before "from django.shortcuts import render, redirect"](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=5459956)